src/components/Tooltip/BaseTooltip.js

```diff
@@ -19,8 +19,42 @@ const hasHoverSupport = DeviceCapabilities.hasHoverSupport();
  * @param {propTypes} props
  * @returns {ReactNodeLike}
  */
+
+/**
+ * Choose the correct bounding box for the tooltip to be positioned against.
+ * This handles the case where the target is wrapped across two lines, and
+ * so we need to find the correct part (the one that the user is hovering
+ * over) and show the tooltip there.
+ *
+ * @param {Element} target The DOM element being hovered over.
+ * @param {number} clientX The X position from the MouseEvent.
+ * @param {number} clientY The Y position from the MouseEvent.
+ * @return {DOMRect} The chosen bounding box.
+ */
+
+function chooseBoundingBox(target, clientX, clientY) {
+    const slop = 5;
+    const bbs = target.getClientRects();
+    const clientXMin = clientX - slop;
+    const clientXMax = clientX + slop;
+    const clientYMin = clientY - slop;
+    const clientYMax = clientY + slop;
+
+    for (let i = 0; i < bbs.length; i++) {
+        const bb = bbs[i];
+        if (clientXMin <= bb.right && clientXMax >= bb.left && clientYMin <= bb.bottom && clientYMax >= bb.top) {
+            return bb;
+        }
+    }
+
+    // If no matching bounding box is found, fall back to the first one.
+    // This could only happen if the user is moving the mouse very quickly
+    // and they got it outside our slop above.
+    return bbs[0];
+}
+
 function Tooltip(props) {
-    const {children, numberOfLines, maxWidth, text, renderTooltipContent, renderTooltipContentKey} = props;
+    const {children, numberOfLines, maxWidth, text, renderTooltipContent, renderTooltipContentKey, shouldHandleScroll} = props;
```

Comment:
Move this to the function parameter 

```js
function Tooltip({children, numberOfLines, maxWidth, text, renderTooltipContent, renderTooltipContentKey, shouldHandleScroll}) {
```

Reply:
We have two lines of code using props and underscore.
`shiftHorizontal={_.result(props, 'shiftHorizontal')}`
`shiftVertical={_.result(props, 'shiftVertical')}`


Should I remove the use of underscore and pass them directly?

or could we have something like this?
```function Tooltip(props, {children, numberOfLines, maxWidth, text, renderTooltipContent, renderTooltipContentKey, shouldHandleScroll}) {```

Reply:
For those two props use `Str.result` (from expensify-common)

```js
shiftHorizontal={Str.result(shiftHorizontal)}
shiftVertical={Str.result(shiftHorizontal)}
```